backend/utils/feed.utils.js

```diff
@@ -0,0 +1,158 @@
+const {
```

Comment:
- this should be a controller not a util if its returning a `response.json`. you can move some and name maybe `userController`.
- util should not contain complex or multiple logic e.g. conditioning. Util should be modular (isolated function) e.g. `transformToUppercase()`, `filterAlphaNumeric()`, etc.